molgenis/molgenis-emx2
# test(rdf): RDF API test (WebApiSmokeTests) in IDEA failing

What are the main changes you did:
`WebApiSmokeTests.testRdfApi()` failed locally with a `java.lang.NoSuchFieldError` error at https://github.com/molgenis/molgenis-emx2/blob/81f9bdd047cd650e93497e25ec37899d05a90aa1/backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java#L127
Conflicting versions could be the issue: [source1](https://groups.google.com/g/rdf4j-users/c/XHEb_cZU-Xc), [source2](https://stackoverflow.com/a/41880423). Though Gradle seems to update older versions already according to `gradle dependencies`, assumption is that something goes wrong when trying to run this test in IDEA.
```
     |    +--- project :backend:molgenis-emx2-graphgenome
[...]
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-api:4.3.12 -> 5.0.1 (*)
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-jsonld:4.3.12 -> 5.0.1 (*)
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-turtle:4.3.12 -> 5.0.1 (*)
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-n3:4.3.12 -> 5.0.1 (*)
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-trig:4.3.12 -> 5.0.1 (*)
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-rdfxml:4.3.12 -> 5.0.1 (*)
     |    |    +--- org.eclipse.rdf4j:rdf4j-rio-ntriples:4.3.12 -> 5.0.1 (*)
     |    |    \--- org.eclipse.rdf4j:rdf4j-rio-nquads:4.3.12 -> 5.0.1 (*)
```

IMPORTANT: After restarting my system, I could not reproduce the issue anymore (maybe would need to do a fresh git clone to reproduce?).


how to test:
- Make sure it runs as usual
- `WebApiSmokeTests.testRdfApi()` should not fail when running it in IDEA (don't forget to add the JVM option `--add-opens=java.base/java.util=ALL-UNNAMED` when testing!).

todo:
